auto_submit/lib/service/validation_service.dart

```diff
@@ -146,20 +145,112 @@ class ValidationService {
       }
     }
     // If we got to this point it means we are ready to submit the PR.
-    bool processed = await processMerge(config, result, messagePullRequest);
-    if (processed) await pubsub.acknowledge('auto-submit-queue-sub', ackId);
-    log.info('Ack the processed message : $ackId.');
+    await processMergeSafely(
+        config, gitHubService, messagePullRequest, result, pubsub, ackId, slug, prNumber, Config.kAutosubmitLabel);
+  }
+
+  /// The logic for processing a revert request and opening the follow up
+  /// review issue in github.
+  Future<void> processRevertRequest(
+    Config config,
+    QueryResult result,
+    gh.PullRequest messagePullRequest,
+    String ackId,
+    PubSub pubsub,
+  ) async {
+    ValidationResult revertValidationResult = await revertValidation!.validate(result, messagePullRequest);
+
+    gh.RepositorySlug slug = messagePullRequest.base!.repo!.slug();
+    final int prNumber = messagePullRequest.number!;
+    final GithubService githubService = await config.createGithubService(slug);
+
+    if (revertValidationResult.result) {
+      try {
+        bool processed = await processMerge(config, result, messagePullRequest);
+        if (processed) {
+          gh.Issue issue = await githubService.createIssue(
+            slug,
+            'Follow up review for revert pull request $prNumber',
+            'Revert request by author ${result.repository!.pullRequest!.author}',
```

Comment:
In it's current state, there is little info. Could we file a P1 to ensure it is triaged, and action steps (such as having someone other than the author close the issue to grant LGTM)

Reply:
Good point. Let me look into this.

Reply:
Okay so there isn't too much you can add to the issue at creation time.
But I did add the following:

required RepositorySlug repositorySlug,
    required String title,
    required String body,
    List<String>? labels,
    String? assignee,
    List<String>? assignees,
    String? state,

I assume the p1 is just a labelbut not sure we can add an action step. It doesn't look like we can add that.